configs/pingcap.json

```diff
@@ -1,11 +1,13 @@
 {
   "index_name": "pingcap",
   "start_urls": [
-    "https://pingcap.com/docs",
+    "https://pingcap.com/docs/",
+    "https://pingcap.com/docs-cn/",
     "https://pingcap.com/"
   ],
   "stop_urls": [
-    "^((?!https://pingcap.com/doc).)*$"
+    "^((?!https://pingcap.com/doc).)*$",
+    "^(https://pingcap.com/doc-)*$",
```

Comment:
Not a valid json because of this extra coma

Reply:
Thank, I'll fix it